e2e/integration_tests/data_flow_test.go

```diff
@@ -123,13 +122,19 @@ func Test_EndpointLinksAreAvailable(t *testing.T) {
 	err = endpoint_orgs_row.Scan(&link_count)
 	failOnError(err)
 
-	if link_count != 2 {
-		t.Fatalf("Database should only have made one link given the fake NPPES data that was loaded. Has: " + strconv.Itoa(link_count))
+	if link_count != 40 {
+		t.Fatalf("Database should only have made 40 links given the fake NPPES data that was loaded. Has: " + strconv.Itoa(link_count))
 	}
 
 	var lantern_org_endpoint_id string
 	err = store.DB.QueryRow("SELECT id FROM fhir_endpoints WHERE organization_name='Lantern Test Org';").Scan(&lantern_org_endpoint_id)
 	failOnError(err)
+	var cape_fear_org_endpoint_id string
```

Comment:
I suggest adding tests for:
an endpoint that maps to multiple NPI organizations
an endpoint that maps to no NPI organizations